Public/Src/Cache/ContentStore/Library/Service/LocalContentServerBase.cs

```diff
@@ -290,10 +290,10 @@ async Task<BoolResult> tryStartupCoreAsync()
             }
         }
 
-        private void InitializeAndStartGrpcServer(int grpcPort, ServerServiceDefinition[] definitions, int requestCallTokensPerCompletionQueue)
+        private void InitializeAndStartGrpcServer(int grpcPort, ServerServiceDefinition[] definitions, int requestCallTokensPerCompletionQueue, int grpcPoolSize = 70)

```

Comment:
nit: the method is called only once, so it is kind of makes no sense to specify the default here.